cirq-core/cirq/value/classical_data.py

```diff
@@ -261,12 +261,16 @@ def _from_json_dict_(
         )
 
     def __repr__(self):
-        return (
-            f'cirq.ClassicalDataDictionaryStore(_records={self.records!r},'
-            f' _measured_qubits={self.measured_qubits!r},'
-            f' _channel_records={self.channel_records!r},'
-            f' _measurement_types={self.measurement_types!r})'
-        )
+        rep = 'cirq.ClassicalDataDictionaryStore('
```

Comment:
Since we're updating the repr format, the corresponding test file (`ClassicalData.repr`) should also be updated.